data/nation/american.ruleset

```diff
@@ -17,6 +17,7 @@ legend=_("The United States of America achieved its independence from\
 leaders = {
  "name",                        "sex"
  "George Washington",           "Male"
+ "Martha Washington",    	"Female"
```

Comment:
Can we try to keep the nice alignment intact?

Reply:
I would say remove Hamilton as he was never a president.